src/components/visualizations/chart/legend/helpers.ts

```diff
@@ -404,6 +408,54 @@ function separateLegendItems(series: any[]) {
     });
 }
 
+export function groupSeriesItemsByType(series: ISeriesItem[]) {
+    const primaryType = get(series, '0.type');
```

Comment:
use [head](https://lodash.com/docs/4.17.11#head) instead of get

Reply:
done